packages/strapi-plugin-graphql/services/utils.js

```diff
@@ -169,6 +169,43 @@ const getActionDetails = resolver => {
 
 const isResolvablePath = path => _.isString(path) && !_.isEmpty(path);
 
+const getDisabledResolverMethods = (resolvers, type) => {
+  const queriesOrMutationResolvers = resolvers[type];
+  const disabledResolverMethods = Object.keys(
+    _.pickBy(queriesOrMutationResolvers, resolver => !resolver)
+  );
+  return disabledResolverMethods;
+};
+
+const hasParameters = methodString => {
+  const hasParametersRegex = /\(([^)]+)\)/;
+  return hasParametersRegex.test(methodString);
+};
+
+const getMethodName = methodString => {
+  const regex = hasParameters(methodString) ? /[^(]*/g : /[^:]*/g;
+  return methodString.match(regex)[0];
+};
+
+const removeDisabledResolvers = (mutationOrQueryDef, disabledResolvers) => {
```

Comment:
I feel like on that string processing could be avoided by using a schema transform on the makeExecutableSchema instead :) this would make it more powerfull too :) inspiration https://www.advancedgraphql.com/content/schema-transformation. 
What do you think ?

Reply:
yes good one! i'll update the code

Reply:
FYI you should use @graphql-tools for this :) they have a lot of utilities to do that